MapboxNavigation/NavigationViewController.swift

```diff
@@ -124,8 +124,10 @@ public class NavigationViewController: NavigationPulleyViewController, RouteMapV
      on the destination of your route. The last coordinate of the route will be
      used if no destination is given.
     */
+    @available(*, deprecated, message: "Destination is no longer support nor necessary. A destination annotation will automatically be added to map given the route.")
```

Comment:
Ah, I missed that we’re exposing `navigationMapView(_:imageFor:)`. In any case, that method will continue to work; it just won’t be possible to say something like `if annotation == navigationViewController.destination`.